# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../glademm/gcc41.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -Nur glademm-2.6.0-orig/src/writers/gnome_app.hh glademm-2.6.0/src/writers/gnome_app.hh
--- glademm-2.6.0-orig/src/writers/gnome_app.hh	2002-04-02 15:14:58.000000000 +0000
+++ glademm-2.6.0/src/writers/gnome_app.hh	2006-04-05 10:43:52.735742750 +0000
@@ -28,7 +28,7 @@
 	virtual bool NeedExplicitCtor(const Widget &w) const;
 	virtual void ConstructionArgs(const Widget &w, CxxFile &f) const;
 	virtual void Configure(const Widget &w, CxxFile &f,const std::string &instance) const;
-	virtual Subwidget Gnome_App::IsSubwidget(const Widget &w,const Widget &ch) const;
+	virtual Subwidget IsSubwidget(const Widget &w,const Widget &ch) const;
 	virtual const std::string InternalInstance(const Widget &parent,const Widget &w2) const;
 	virtual void AddChildren(const Widget &w,CxxFile &f,const std::string &instance,const WriterBase &writer_for_subw, const Widget &widget_for_subw) const;
 };
